src/ws/cockpitwebservice.c

```diff
@@ -1116,7 +1116,10 @@ on_web_socket_open (WebSocketConnection *connection,
   JsonObject *object;
   JsonObject *info;
 
-  g_info ("New connection to session from %s", cockpit_creds_get_rhost (self->creds));
+  if (cockpit_creds_get_rhost (self->creds))
+      g_info ("New connection to session from %s", cockpit_creds_get_rhost (self->creds));
+  else
+      g_info ("New local connection to session");
```

Comment:
I would just say "New connection to session".  "local" is misleading.

Reply:
Ack, changed.